tests/unit/lib/test_feature_utils.py

```diff
@@ -64,11 +64,6 @@ def test_not_array_value_error():
         dc.read_values(value=True)
 
 
-def test_empty_value_list_error():
-    with pytest.raises(ValuesToTupleError):
-        dc.read_values(value=[])
```

Comment:
Dropped the test. Now the column type will default to `Optional[str]` in case of empty values.